conf/virtwho.yaml.template

```diff
@@ -0,0 +1,23 @@
+VIRTWHO:
```

Comment:
@ColeHiggins2 and I discussed the scope of this file yesterday, and this PR will need some significant changes.

virtwho.properties is what is actually utilized by the virtwho tests, the VirtwhoSettings class that is defined, and that has this structure in robotteo.properties, is not in use by any of the tests or framework.

This file should have a structure where there are hypervisor details under a map for each hypervisor type, like:

```
VIRTWHO:
  ESX:
    SERVER:
    USERNAME:
    GUEST:
    ETC:
  XEN:
    SERVER:
    USERNAME:
    GUEST:
    ETC:
```